src/babel-benchmark.js

```diff
@@ -2,14 +2,27 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-const Babel = require("@babel/standalone");
+const Babel = require("@babel/core");
```

Comment:
Surprised this would work? I thought the point was that we were using standalone since it was run with webpack. We can use `@babel/standalone` and `@babel/preset-env-standalone`

Reply:
Why is it weird that this works? :D What would be the benefit of using `@babel/standalone` + `@babel/preset-env-standalone`?

Reply:
Thought that the environment for testing didn't recognize requiring modules before, if it works then it's certainly closer to what would be run so never mind

Reply:
@hzoo Do you mean something like this?

```diff
diff --git a/package.json b/package.json
index 460277a..da42db1 100644
--- a/package.json
+++ b/package.json
@@ -33,8 +33,8 @@
   },
   "license": "BSD-3-Clause",
   "dependencies": {
-    "@babel/core": "^7.0.0-beta.34",
-    "@babel/preset-env": "^7.0.0-beta.34",
+    "@babel/preset-env-standalone": "^7.0.0-beta.35",
+    "@babel/standalone": "^7.0.0-beta.35",
     "acorn": "5.2.1",
     "babylon": "7.0.0-beta.34",
     "benchmark": "^2.1.4",
diff --git a/src/babel-benchmark.js b/src/babel-benchmark.js
index 52109f3..0331a0d 100644
--- a/src/babel-benchmark.js
+++ b/src/babel-benchmark.js
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-const Babel = require("@babel/core");
+const Babel = require("@babel/standalone");
 const babylon = require("babylon");
 const fs = require("fs");
 
@@ -12,7 +12,7 @@ const payloads = [
     options: {
       presets: [
         [
-          "@babel/preset-env",
+          "@babel/preset-env-standalone",
           {
             targets: {
               browsers: ["last 2 versions"],
```

I’m getting `Invalid preset specified in Babel options: "@babel/preset-env-standalone,[object Object]"` when I do that.